tools/dockerfile/interoptest/grpc_interop_cxx_xds/Dockerfile.xds_server

```diff
@@ -38,5 +45,12 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 COPY --from=0 /artifacts ./
+COPY --from=0 /workdir/grpcurl /bin/grpcurl
+
+# TODO: The reflection server is not currently fully functional. Copying the
+# protos is required for CLI RPC tools run against localhost of this container
+# image until this is fixed.
+COPY --from=0 /workdir/src/proto  /src/proto/
+COPY tools/dockerfile/interoptest/grpc_interop_cxx_xds/xds_server_entrypoint.sh /
 
-ENTRYPOINT ["/xds_interop_server"]
+ENTRYPOINT ["/xds_server_entrypoint.sh"]
```

Comment:
Let's add a todo to replace it with [tini](https://github.com/krallin/tini) or similar.

Reply:
Done.